src/celeritas/field/RZMapFieldInput.hh

```diff
@@ -29,8 +29,10 @@ struct RZMapFieldInput
 {
     unsigned int num_grid_z{};
     unsigned int num_grid_r{};
-    double delta_grid{};  //!< Grid spacing [cm]
-    double offset_z{};  //!< Offset of the lower z coordinate [cm]
+    double min_z{};  //!< Lower z coordinate [cm]
+    double min_r{};  //!< Lower r coordinate [cm]
```

Comment:
Even though it's necessary (from an implementation point of view) to have a lower R coordinate in the RZ data, I don't think there should be a user input for a minimum R value.

Reply:
Actually, there will be a user case that a volume-based field map between r_min (!=0) and r_max is used.  For an example, a parameterized field (or uniform field) can be used in the central detector region, r < r_min  and a fieldmap in the rest.